src/modules/common/components/__tests__/Page.spec.js

```diff
@@ -7,7 +7,7 @@ import Page from '../Page'
 
 describe('Page', () => {
   const title = 'first Event'
-  const lastUpdate = moment('2016-01-07 10:36:24')
+  const lastUpdate = moment('2016-01-07T10:36:24.000Z', moment.ISO_8601)
```

Comment:
```suggestion
  const lastUpdate = moment('2016-01-07T10:36:24.000Z')
```

Reply:
Is this already the default?

Reply:
yes, see https://momentjs.com/docs/#/parsing/string/